docs/general/faq.md

```diff
@@ -28,10 +44,76 @@ UTC.
 On August 21, 2020, Redenomination of DOT occurred. From this date, one DOT (old) equals 100 new
 DOT.
 
-## Polkadot Roadmap
+### Why Polkadot targets 1000 validators while other projects have hundreds of thousands?
 
-For more information on the Polkadot roadmap please visit the
-[official Polkadot website](https://www.opengov.watch/polkadot-roadmap).
+Polkadot's goal to have 1000 validators is set to be something that is practically achievable with high confidence of good performance in a live environment. Furthermore, validators in Polkadot are not the only stakers, and if we consider the number of stakers that can be possible on Polkadot the number can scale up to hundreds of thousands. Since validators are performing critical consensus work to maintain the security of the chain including all of its shards, a more modest number of validators is estimated to start. Upon improvements, such as implementing signature aggregation for finalization messages, the number of validators can reasonably scale up.
+
+Additionally, other projects sometimes have a different definition of _validator_ that approximates more closely to remote signing keys without the full operation of a validating node. On Polkadot, each validator is running their own validating node and performing full verification of the Relay Chain, voting on finality, producing blocks in their decided slots, and verifying parachain state transitions. Other projects may consider validators and "validating nodes" as separate entities.
+
+Finally, individuals may participate in the block production process indirectly by
+[nominating](../learn/learn-nominator.md) validators. In this way, individuals who are not running a
+node can still share in staking rewards.
+
+### What is the block time of the relay chain?
+
+Both the Kusama and Polkadot networks are currently operating at a rate of one block every six
+seconds.
+
+### Does Polkadot have smart contracts?
+
+While the Polkadot relay chain does not implement smart contracts directly, [there are numerous
```

Comment:
```suggestion
Polkadot will support Solidity smart contracts by late 2025. Smart contracts deployment is already possible on Kusama and testnets. There are also numerous parachains that support EVM and Wasm smart contracts.
```

Reply:
Yeah given the importance of this feature I would add it